projects/plugins/crm/includes/ZeroBSCRM.Core.php

```diff
@@ -2530,7 +2530,7 @@ public function setPageTitle( $title = '', $adminTitle = '' ) {
 		$pageTitle = ( ( $adminTitle == '' ) ? __( 'Jetpack CRM', 'zero-bs-crm' ) : $adminTitle );
 
 		// useful? global $post, $title, $action, $current_screen;
-		// global $zbsPage; print_r($zbsPage); exit();
+		// global $zbsPage; print_r($zbsPage); exit( 0 );
```

Comment:
Same here, I think the comment has to go instead.

Reply:
460bd03e28